packages/cosmic-proto/scripts/codegen.cjs

```diff
@@ -86,13 +86,13 @@ telescope({
     },
     aminoEncoding: {
       // Necessary for getSigningAgoricClient
-      enabled: true,
+      enabled: false,
```

Comment:
`dapp-inter` and other web clients are likely dependent on amino encodings. Agree they should be removed for `vats`, but something to make sure is in scope as part of #9200

Reply:
Good callout, though it won't break until they try to upgrade past 0.4.0 and by then we may have #9200. If not then they shouldn't upgrade. I expect their CIs to fail if they upgrade to a version lacking the requisite functionality.